components/Tooltip/Tooltip.jsx

```diff
@@ -43,17 +43,18 @@ const Tip = styled.div`
 
 const TipText = styled.span`
   display: inline-block;
-  max-width: 250px;
   overflow: hidden;
   text-overflow: ellipsis;
-  white-space: nowrap;
+  white-space: pre;
   line-height: 20px;
+  text-align: left;
```

Comment:
This line has been added to left align the text.

Reply:
what do you think to set this property using theme? That's can be used to _left_ or _center_ text.